scripts/test_Change_Log_info/README.md

```diff
@@ -0,0 +1,12 @@
+# Overview
+This script will use docker and help you generate two code_reports which generate by the latest two versions of azure MGMT package on pypi..
+And then it will automatically use change_ log for these two code_reports.
+Finally , it will generate a folder in the current directory, and the results of each comparison will be stored separately in the folder in the form of TXT.
+
```

Comment:
Add `# preprequisites` like 
https://github.com/Azure/azure-sdk-for-python/wiki/Dataplane-Codegen-Quick-Start#prerequisites) to give more details about what need before running the tool